website/blog/2022-09-05-version-070.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Announcing React Native 0.70
+authors: [dmitryrykun, titozzz, cortinico, kelset]
+tags: [announcement, release]
+---
+
+# Announcing 0.70
+
+We are excited to release a new version of React Native, 0.70.0. This version comes with several improvements like a new unified configuration for Codegen, Hermes as default engine, and full CMake support for Android builds along with a refresh of the documentation for the New Architecture. Read on to learn more!
+
+### Sections
+
+- [New Architecture’s New Documentation](/blog/2022/09/05/version-070#new-architectures-new-documentation)
+- [Hermes as default engine](/blog/2022/09/05/version-070#hermes-as-default-engine)
+- [A new unified configuration for Codegen](/blog/2022/09/05/version-070#a-new-unified-configuration-for-codegen)
+- [Android Auto-linking for New Architecture libraries](/blog/2022/09/05/version-070#android-auto-linking-for-new-architecture-libraries)
+- [Full CMake support for Android builds](/blog/2022/09/05/version-070#full-cmake-support-for-android-builds)
+- [Highlights of 0.70](/blog/2022/09/05/version-070#highlights-of-070)
+
+<!--truncate-->
+
+## New Architecture’s New Documentation
+
+Over the last few months we have been working to add more content to the [New Architecture](https://reactnative.dev/docs/next/the-new-architecture/landing-page) section of the documentation: in the new section you will find migration guides, examples and tutorials to get you up to speed.
+
+Along with it, you will find new documents diving into [Why a New Architecture](https://reactnative.dev/docs/next/the-new-architecture/why) and [the various parts of it](https://reactnative.dev/docs/next/the-new-architecture/pillars), that we hope will help you better understand the reasoning around the new APIs.
+
+Any feedback is more than welcome, please let us know in the [react-native-website](https://github.com/facebook/react-native-website) repository.
+
+## Hermes as default engine
+
+React Native 0.70 will be the first version of having Hermes, our in-house JS engine, enabled by default.
+
+This is the result of collaborative effort between the Hermes team and the React Native team, alongside with the priceless contributions from the community. We worked to improve and fine tune Hermes to make it more performant and deliver highly requested features by the community.
+
+You can read more about it in the [official announcement blogpost](https://reactnative.dev/blog/2022/07/08/hermes-as-the-default).
```

Comment:
Is it worth to say something about Apple accepting Hermes as JS engine instead of JSC?

Reply:
Not really as it gets really offtopic otherwise.